### PR TITLE
Konvert ProtoFileElement

### PR DESCRIPTION
**Pull request by jrodbx**

### CODE REVIEW

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java [hunk 1]
```diff
@@ -98,7 +101,6 @@ private Object readDeclaration(String documentation, Context context) {
       if (!context.permitsPackage()) throw reader.unexpected(location, "'package' in " + context);
       if (packageName != null) throw reader.unexpected(location, "too many package names");
       packageName = reader.readName();
-      fileBuilder.packageName(packageName);
```
Comment (Egorand) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java]: Is it safe to just delete this line? Probably should do `location.copy()` and replace the package name?

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java [hunk 2]
```diff
@@ -538,7 +540,7 @@ private RpcElement readRpc(Location location, String documentation) {
     reader.require(')');
 
     if (reader.peekChar('{')) {
-      ImmutableList.Builder<OptionElement> options = ImmutableList.builder();
+    ImmutableList.Builder<OptionElement> options = ImmutableList.builder();
```
Comment (Egorand) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java]: Revert?

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java (hunk 1 above)
Comment (jrodbx) [reply wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java]: See the other changes in this file.

Comment (Egorand) [reply wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java]: Ah, got it, nvm.

**Pull request merged**